### PR TITLE
Replace feed cypress test with equivalent pytest

### LINKED ISSUE #394: Replace cypress feed.cy.js with PyTest integration tests.

### PR DESCRIPTION
**Pull request by brianpeiris**
### Detailed information:

- Deleted feed.cy.js and replaced it with an equivalent pytest in app/feed/tests/test_feed_routes.py
- Removed obsolete feedEndpoint cypress command.

### Closing issues: 

List all issues the pull request solve: 

- Closes #394

### Test plan (required)

- [x] The PR contains new PyTest unit/integration tests for any function or functional added. 
- [x] The PR changes existing PyTest unit/integration tests to keep all tests up to date.
- [x] The PR does not lead to degradation in unit test coverage.
- [x] Project parts affected by changes in this PR was tested manually on your local (using Postman or in any other way). List everything you've tested below:
  - ran all pytests

### CODE REVIEW

File: app/feed/tests/test_feed_routes.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from app.common.tests.utils import is_none_or_type
+from app.factories import ScoresFactory
+
+
+@pytest.mark.integration
+def test_feed(client_with_user_and_header):
+    client, _, session_header, _ = client_with_user_and_header
+    score = ScoresFactory()
+    response = client.get(
+        url_for("scoring.get_feed"),
```
Comment (brianpeiris) [inline app/feed/tests/test_feed_routes.py:13]: I would have expected this to be "feed.get_feed", but `feed/routes.py` does actually use the Blueprint from `app.scoring` https://github.com/ClimateMind/climatemind-backend/blob/develop/app/feed/routes.py#L4

File: conftest.py [hunk 2]
```diff
@@ -61,7 +61,6 @@ def pytest_sessionstart(session):
 
 def session_config_has_lrf_data_enabled(session_config):
     marker_option = session_config.getoption("-m")
-    print("marker_option", marker_option)
```
Comment (brianpeiris) [inline conftest.py:64]: Accidentally left this print statement in #466, so I'm just removing it here.

File: app/feed/tests/test_feed_routes.py (hunk 1 above)
Comment (danmash) [reply app/feed/tests/test_feed_routes.py:14]: @brianpeiris I'll create a separate issue

**Review by danmash — APPROVED** [review]
Thanks! I'm glad we were figured out with lrf_data.

**Pull request merged**